scripts/release/update_dois/setup_update_dois.pl

```diff
@@ -0,0 +1,43 @@
+#!/usr/local/bin/perl  -w
+use strict;
+
+use lib '/usr/local/gkb/modules';
+# use GKB::Config;
+
+use autodie qw/:all/;
+use Cwd;
+use Getopt::Long;
+use Data::Dumper;
+use File::Basename;
+use List::MoreUtils qw/uniq/;
+
+use Log::Log4perl qw/get_logger/;
+# Log::Log4perl->init(\$LOG_CONF);
+my $logger = get_logger(__PACKAGE__);
+
+my $arg;
+if ($ARGV[0] && $ARGV[0] eq '-clone') {
+  $arg = $ARGV[0];
+}
+
+if (-e "data-release-pipeline" || !$arg) {
+  chdir "data-release-pipeline/update-dois";
```

Comment:
Maybe a good idea to do a `git pull` to get the latest changes, when the directory already exists.